build-tools/packages/build-cli/src/repoConfig.ts

```diff
@@ -3,7 +3,28 @@
  * Licensed under the MIT License.
  */
 
+import minimatch from "minimatch";
+import { ReleaseGroup, ReleasePackage } from "./releaseGroups";
+
+// Mapping of branch to a list of release groups/packages that should run policy by default.
+// TODO: This should be configured in the fluid-build config, like type test defaults are.
+const defaults = {
+	"main": ["client"],
+	"release/*": ["client"],
+};
+
 /**
- * The branches in this list should run policy check by default during release-related activities.
+ * Returns true if policy-check should run by default for a given branch/release group combination.
  */
-export const branchesRunDefaultPolicy = ["main"];
+export const getPolicyRunDefault = (
```

Comment:
I find the name of this function confusing. I think "RunPolicy" makes more sense as the name than "PolicyRun", and shorting policy-check to policy makes it less clear.

`getRunPolicyCheckDefault` seems better.

Reply:
Agreed -- will update.

Reply:
Resolving comments before pushing fixes makes code review harder

Reply:
nervermind, just github having a bad UI? I can't event tell if you fixed this easilly.